alert-engine/src/test/scala/io/phdata/pulse/alertengine/AlertEngineImplTest.scala

```diff
@@ -78,11 +93,13 @@ class AlertEngineImplTest extends FunSuite with BaseSolrCloudTest with MockitoSu
       new AlertEngineImpl(
         solrClient,
         new NotificationServiceFactory(mailNotificationService, slackNotificationService))
-    val result = engine.triggeredAlert(TEST_COLLECTION, alert).get
+    val result: TriggeredAlert = engine.triggeredAlert(TEST_COLLECTION, alert).get
```

Comment:
You can remove this type

Reply:
Ok, I will remove this one, thanks for catching this!